pom.xml

```diff
@@ -246,6 +246,11 @@
         <artifactId>slf4j-log4j12</artifactId>
         <version>${slf4j.version}</version>
       </dependency>
+      <dependency>
```

Comment:
I think the convention is to avoid adding the dependency to modules that do not need it (which would happen if you make it a top-level depedency).  underfs/pom.xml is the appropriate place for it